CONTRIBUTING.md

```diff
@@ -58,15 +61,25 @@ go build
 **Running the linter**:
 
 We make use of the [golangci-lint](https://github.com/golangci/golangci-lint) tool to lint the code in CI. To run it locally, first install it:
+
 ```bash
 go get -u github.com/golangci/golangci-lint/cmd/golangci-lint
 ```
+
 then run:
+
+```bash
+make lint
 ```
-golangci-lint run --out-format=tab --new-from-rev master ./...
+
+You can also run the linter inside the docker container, which will benefit from the version of the linter being the same as it will be in CI.
+
+```bash
+make ci-like-lint
 ```
 
-If you've added new dependencies you might also want to check and make sure all dependencies exists in `vendor/` folder by running:
+If you've added new dependencies (and just in case [we have a policy about it](Dependencies.md)) you might also want to check and make sure all dependencies exists in `vendor/` folder by running:
+
 ```bash
 go get -u github.com/FiloSottile/vendorcheck
 vendorcheck ./...
```

Comment:
lol this can probably go away ... It has been deprecated by the author and if they will be doing something they can just as well do `go mod vendor` which will even fix it for them.